new_dawn_server/pusher/notification_service.py

```diff
@@ -0,0 +1,33 @@
+from pusher_push_notifications import PushNotifications
+
+beams_client = PushNotifications(
+    instance_id='466cb5de-0fd2-40c7-ad45-802b6c79550e',
+    secret_key='26C789164E94F87D4125293F36DE3C988A36998FBEF78FE8BFE86B274E998BFD',
```

Comment:
Put this as a constant into local .env file (create one if not existed in new_dawn_server/). Get this key from settings.py.  

You can search google for dotenv for how this works.